src/files-and-videos/videos/data/api.test.js

```diff
@@ -0,0 +1,55 @@
+import { getDownload } from './api';
+import 'file-saver';
+
+jest.mock('file-saver');
+
+describe('api.js', () => {
+  describe('getDownload', () => {
```

Comment:
The tests are great, but I only see tests for the `getDownload` function. It would be good to test each function in here directly or at least indirectly when they're called in another function. Covering them by testing the react components that call them I think is not be enough to cover edge cases - you see from the tests for this function alone how many possible edge cases there are.

I don't think this needs to account for every possible edge case at all, but just the most important ones.

Reply:
I think that only the `getDownload` has a lot of edge cases because of the nested errors. I think that I sufficiently cover the cases in the integration tests for the other api calls.